stream-chat-android-ui-components/src/main/res/layout/stream_fragment_attachment_camera.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why not just simple FrameLayout? Is constraint required here?

Reply:
Lets leave it as is for now, `stream_attachment_permission_view.xml` include is used in different fragment layouts. And the included layout relies on the fact that it is included into `ConstraintLayout`.